project/login.html

```diff
@@ -130,21 +130,40 @@
 				margin-left: 1400px;
 			}
 
+			::-webkit-input-placeholder { /* WebKit, Blink, Edge */
+			    color: lightgrey;
+			}
+			:-moz-placeholder { /* Mozilla Firefox 4 to 18 */
+			   color: lightgrey;
+			   opacity:  1;
+			}
+			::-moz-placeholder { /* Mozilla Firefox 19+ */
+			   color: lightgrey;
+			   opacity:  1;
+			}
+			:-ms-input-placeholder { /* Internet Explorer 10-11 */
+			   color: lightgrey;
+			}
+			::-ms-input-placeholder { /* Microsoft Edge */
+			   color: lightgrey;
+			}
+
+
 		</style>
 	</head>
 
 	<body>
 
-		<div class="firstBox">
+		<div style="opacity:0.8" class="firstBox">
```

Comment:
I wanted to be faster at the time, but I will move it to an external file, together with all the css in <style> because it is too much code to keep it in the head.